### PR TITLE
Don't allow renegotiation for DTLS 1.3

### PR DESCRIPTION
**Pull request by fwh-dc**

### CODE REVIEW

**Review by t8m — APPROVED** [review]

File: apps/include/s_apps.h [hunk 1]
```diff
@@ -16,7 +16,8 @@
 #define PROTOCOL        "tcp"
 
 #define SSL_VERSION_ALLOWS_RENEGOTIATION(s) \
-    (SSL_is_dtls(s) || (SSL_version(s) < TLS1_3_VERSION))
+    ((SSL_is_dtls(s) && SSL_version(s) > DTLS1_3_VERSION) \
```
Comment (mattcaswell) [inline apps/include/s_apps.h:19]: What about DTLS1_BAD_VER?

Comment (fwh-dc) [reply apps/include/s_apps.h:19]: Good catch!

**Review by mattcaswell — APPROVED** [review]

File: apps/include/s_apps.h [hunk 2]
```diff
@@ -16,7 +16,9 @@
 #define PROTOCOL        "tcp"
 
 #define SSL_VERSION_ALLOWS_RENEGOTIATION(s) \
-    (SSL_is_dtls(s) || (SSL_version(s) < TLS1_3_VERSION))
+    ((SSL_is_dtls(s) && (SSL_version(s) > DTLS1_3_VERSION \
+                         || SSL_version(s) == DTLS1_BAD_VER) \
```
Comment (mattcaswell) [inline apps/include/s_apps.h:20]: paren mismatch here??

**Review by mattcaswell — CHANGES_REQUESTED** [review]

**Review by t8m — APPROVED** [review]

**Review by mattcaswell — APPROVED** [review]
CI failure is not relevant

**Pull request closed without merging**